.gitmodules

```diff
@@ -4,3 +4,11 @@
 [submodule "src/instrumentation_measurement/evmone"]
 	path = src/instrumentation_measurement/evmone
 	url = ../evmone.git
+[submodule "src/instrumentation_measurement/geth_benchmark"]
+	path = src/instrumentation_measurement/geth_benchmark
+	url = https://github.com/JacekGlen/go-ethereum
```

Comment:
pls move the 2 branches into `imapp-pl` org forks

Reply:
also pls make sure to branch off the same commit as `wallclock` does

Reply:
and (3.) pls create non-mergeable PRs so that I can see the diff vs that SHA in `imapp-pl/master`

Reply:
Branches updated and links to non-mergeable PRs added